lib/Cavil/Checkout.pm

```diff
@@ -148,9 +151,19 @@ sub unpack {
   $u->mime_helper_dir('/usr/share/File-Unpack/helper/');
   $u->mime_helper_dir($MIME);
   eval { $u->unpack($dir) };
-  return
-    unless my $err = $@ || ($u->{error} ? join(', ', @{$u->{error}}) : undef);
-  -f $log ? warn $err : die $err;
+  my $err = $@ || ($u->{error} ? join(', ', @{$u->{error}}) : undef);
+
+  if ($err) {
+    -f $log ? warn $err : die $err;
+    return;
+  }
+
+  my $unpacked  = decode_json($dir->child('.unpacked.json')->slurp);
+  my $processor = Cavil::PostProcess->new($unpacked);
+  $processor->postprocess;
+  open(my $fh, '>', $dir->child('.postprocessed.json'));
+  print $fh encode_json($processor->hash);
+  close($fh);
```

Comment:
[Mojo::File::spurt](https://metacpan.org/pod/Mojo::File#spurt) is more efficient for writing a whole file at once. (`$dir->child('.postprocessed.json')->spurt(encode_json($processor->hash))`)